### PR TITLE
Correct text for Amazon EMR and Ambari

### PR DESCRIPTION
**Pull request by awholegunch**
Fix for https://issues.cask.co/browse/CDAP-8002

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/index.rst [hunk 1]
```diff
@@ -30,16 +30,15 @@ Installation and configuration instructions for either **specific distributions*
...
+    Explore engines, and enabling Kerberos) are currently not available for Amazon EMR
 
 ..
 
 - :ref:`Installation using Apache Ambari <admin-installation-ambari>`
 
   - :ref:`Advanced Topics: <ambari-installation-advanced-topics>`
-    enabling Kerberos; enabling perimeter security, CDAP high availability, and CDAP Explore engines
-    are currently not available for Ambari
+    enabling perimeter security, Kerberos, CDAP high availability, and CDAP Explore engines
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/installation/index.rst:41]: Remove the reference to explore... it's enabled/configured by default.

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
One comment

File: cdap-docs/admin-manual/source/installation/index.rst (hunk 1 above)
Comment (awholegunch) [reply cdap-docs/admin-manual/source/installation/index.rst:41]: Does that mean that the text at the link (http://docs.cask.co/cdap/current/en/admin-manual/installation/ambari.html#hive-execution-engines) needs to be changed?

Comment (awholegunch) [reply cdap-docs/admin-manual/source/installation/index.rst:41]: Perhaps say "configuring CDAP Explore engines"?

Comment (wolf31o2) [reply cdap-docs/admin-manual/source/installation/index.rst:41]: Oh... the engines! My bad.

**Review by wolf31o2 — APPROVED** [review]

**Comment by wolf31o2** [comment]
👍 LGTM

**Pull request merged**